server/mediaserver_test.go

```diff
@@ -158,7 +158,9 @@ func TestCreateRTMPStreamHandler(t *testing.T) {
 	// (Would be nice to stub out parseManifestID to receive stronger
 	//  transitive assurance via existing parseManifestID tests)
 	testHlsQueryParam := func(inp string) {
```

Comment:
Should this be renamed to `testManifestIDQueryParam`?

Reply:
Fixed